src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx

```diff
@@ -72,7 +73,12 @@ const HeaderContainer = ({
                 setErrorMessage(error.message);
             })
             .then(({formTitle, pageSlug}: FormSettings) => {
-                dispatch(setFormSettings({formTitle, pageSlug}));
+                dispatch(
+                    setFormSettings({
+                        formTitle,
+                        pageSlug: pageSlug,
```

Comment:
```suggestion
                        pageSlug,
```

Reply:
Meant to revert that. Thanks! Resolved: d023df110da7d361d8c912dced4a6999e18b98f1